### ISSUE TITLE
Generate cryptographically secure random numbers

### ISSUE THREAD

**Issue by cycomachead** [bromagosa/snapCloud#14]
https://github.com/bromagosa/snapCloud/blob/master/api.lua#L120

It seems like we should be using something a bit stronger for a salt.
https://github.com/openresty/lua-resty-string seems like a good bet, and the readme has the following example
```
 local resty_random = require "resty.random"
    local str = require "resty.string"
    local strong_random = resty_random.bytes(16,true)
        -- attempt to generate 16 bytes of
        -- cryptographically strong random data
    while strong_random == nil do
        strong_random = resty_random.bytes(16,true)
    end
    ngx.say("random: ", str.to_hex(strong_random))
```

**Comment by bromagosa** [comment]
can you send a PR?

**Comment by bromagosa** [comment]
Thanks Michael! Fixed in your PR #17
